Session2_Intro_To_Machine_Learning_For_Meteorological_Science/01_NASA_Space_Apps_Bootcamp_MLP_with_Jena_data.ipynb

```diff
@@ -0,0 +1,2231 @@
+{
```

Comment:
<p>If you would like, you can probably remove the CoLab disclaimer</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/informatics-lab/spaceapps-2022-mo-bootcamp/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='69'/>

Reply:
I've left this in for now until I speak to Anna later and work out what I'll replace it with